src/hooks/useInfiniteScroll.tsx

```diff
@@ -0,0 +1,25 @@
+import { useEffect } from 'react'
+
+const useInfiniteScroll = (enabled: boolean, handleReachBottom: () => void) => {
+  useEffect(() => {
+    if (!enabled) return
+
+    const handleScroll = () => {
+      if (
+        window.innerHeight + document.documentElement.scrollTop >=
+        document.documentElement.offsetHeight
+      ) {
+        console.log('Scroll')
```

Comment:
```suggestion
```

Reply:
Yeah, spotted that when trying to improve tests :see_no_evil: